core/common/src/main/java/alluxio/Configuration.java

```diff
@@ -281,9 +276,9 @@ public static double getDouble(String key) {
    * @param key the key to get the value for
    * @return the value for the given key as a {@code float}
    */
-  public static float getFloat(String key) {
-    if (PROPERTIES.containsKey(key)) {
-      String rawValue = PROPERTIES.getProperty(key);
+  public static float PropertyKey(PropertyKey key) {
```

Comment:
I think andrew has a PR rewriting tests of configuration. I will merge with that later.